core/project_settings.cpp

```diff
@@ -1044,6 +1045,10 @@ ProjectSettings::ProjectSettings() {
 	custom_prop_info["compression/formats/gzip/compression_level"] = PropertyInfo(Variant::INT, "compression/formats/gzip/compression_level", PROPERTY_HINT_RANGE, "-1,9,1");
 
 	using_datapack = false;
+
+	// Physics settings
+	Compression::zstd_level = GLOBAL_DEF(PhysicsServerManager::setting_property_name, "DEFAULT");
```

Comment:
this doesnt make any sense

Reply:
Do you mean that the assignment "Compression::zstd_level = GLOBAL_DEF" is useless or the definition of "DEFAULT"?

My idea was to allow the developer to set the default physics server directly in the code. So eventually is not required by the "Godot" user that change the settings in order to use the default physics.

Reply:
What's the link with `Compression::zstd_level` though?